aip/0002.md

```diff
@@ -11,45 +11,54 @@ redirect_from:
   - /0002
 ---
 
-# AIP Numbering
+# AIP Identifiers
 
 The AIP system provides a mechanism to index and have a single source of truth
 for API Improvement Proposals, as well as iterate on them collaboratively and
 transparently.
 
-This document describes the AIP numbering system.
+Multiple companies and teams have adopted the AIP system, with the mutual goal
+that the identifiers for common concepts should be shared between each
+company's AIP program to provide comparability.
 
 ## Assigning AIP Numbers
 
-The AIP editors (see [AIP-1](./0001.md)) are responsible for assigning a number
-to each AIP when it is accepted as a draft for review. Importantly, _all_ AIPs
-have numbers, not just approved ones. The [AIP Index](/) clearly delineates
-which AIPs are approved and binding and which are under discussion.
+The AIP editors (see [AIP A-1](./0001.md)) are responsible for assigning a
+unique identifier to each AIP when it is accepted as a draft for review. For
+AIPs expected to be common between companies, a common prefix (either one or
+two letters) and one number are assigned, and shared across the system. AIPs
+specific to a single company are given a three-letter prefix instead.
+Three-letter prefixes are only guaranteed to be unique within a given company.
 
-The editors may decide to reserve a specific block of numbers for groups of
-AIPs that are related in some way (for example, that are only scoped to a
-specific subset of APIs).
-
-Beyond this, AIP numbers are assigned arbitrarily. In general, the editors will
-take the next AIP number off of the stack to assign to a draft AIP, but
-occasionally may use a special/joke number if useful for mnemonic or other
-reasons.
+Importantly, _all_ AIPs have identifiers, not just approved ones. The
+[AIP Index](/) clearly delineates which AIPs are approved and binding and which
+are under discussion.
 
 ## AIP Blocks
 
-Currently recognized blocks of AIP numbers are:
-
-### Generally Applicable
+Common AIPs are organized into the following over-arching categories:
 
-- **1-99:** Reserved for meta-AIPs (generally process-related).
-- **100-999:** General API design guidance
+- **M:** Meta-AIPs (AIPs about AIPs)
```

Comment:
```suggestion
- **M:** Meta-AIPs (AIPs about AIPs)
-**G:** General
```

Reply:
Suggesting a general category. This would be a great place for things like an Introduction, a Mission Statement, Values, Design Philosophy, etc. For example I think [AIP-2712](https://aip.dev/apps/2712) Could fit under this category.

May also include Versioning, Documentation, Contribution Guidelines, Bad API Precedent if we don't think they belong better under process.

Reply:
Supportive in principle.